### PR TITLE
Fix/5937

### LINKED ISSUE #5937: Date not rendering correctly in auth users page
# Bug report

Dates turn up as "Invalid Date" specifically on Safari browser because of an issue within Dajys:
https://github.com/iamkun/dayjs/issues/1822

Either we wait for a fix from them or we create a cross-browser fix for safari

### PR DESCRIPTION
**Pull request by joshenlim**
Resolves https://github.com/supabase/supabase/issues/5937

Just FYI I've only observed the issue / seen reports of this issue on the Auth users page specifically
Mainly because the user's `created_at` and `last_signed_in_at` has an odd(?) format? - there's a space between the date and time segment, which fails on Safari but chrome and firefox are fine
Hence why i've made the solution within the Auth/Users folder

I did a quick check on the other pages that use `dayjs.format()`, but they don't seem to be broken
- e.g BackupsList component, each backup's `inserted_at` follows the correct format of having `T` in between the date and time segments
- Subscriptions component, its using a numerical timestamp value

### CODE REVIEW

**Comment by saltcod** [comment]
@joshenlim i think this is unrelated to the other date issue, but I wonder if that can be fixed with this one. 

To reproduce: 

1. On any timestampz field, change the seconds to 00
2. Hit save
3. Get the invalid date error.

CleanShot 2022-05-12 at 08 29 00@2x


CleanShot 2022-05-12 at 08 30 39@2x

**Comment by joshenlim** [comment]
Yup the one you mentioned should have been fixed by https://github.com/supabase/supabase/pull/6766

**Comment by joshenlim** [comment]
I'll give it a go locally too actually just to verify

**Comment by joshenlim** [comment]
image
ahh this is annoying - when seconds are 0 the HTML input doesnt return seconds, i'll fix that

File: studio/components/interfaces/Authentication/Users/Users.utils.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ * Formats the user datetime string into a format that's
+ * readable across browsers.
+ * - Postgres: 2022-05-12 09:24:12.639463+00
+ *
+ * These are the acceptable formats that Joshen has experimented so far
+ * - Chrome:   2022-05-12 09:24:12.639463+00 (No format required)
+ * - Firefox:  2022-05-12 09:24:12.639463+00 (No format required)
+ * - Safari:   2022-05-12T09:24:12.639463+00
+ *
+ * Note: I've only seen this issue on the Auth page, hence why keeping
```
Comment (MildTomato) [inline studio/components/interfaces/Authentication/Users/Users.utils.ts:13]: good idea

**Review by MildTomato — APPROVED** [review]

**Pull request merged**